com.unity.cinemachine/Documentation~/samples-tutorials.md

```diff
@@ -2,15 +2,48 @@
 
 Several sample scenes and video tutorials are available in addition to this documentation to demonstrate how to use the Cinemachine features in real-world scenarios.
 
-## Samples
+## Sample scenes
 
 The Cinemachine package includes sample scenes that you can [import to your project](samples-import.md) to discover many Cinemachine features in different contexts of use.
 
-| Sample | Description |
-| :--- | :--- |
-| 2D Samples | Sample scenes illustrating various ways to use Cinemachine in a 2D setting.<br />Includes: `2D Fighters`, `2D Platformer`, `CameraMagnets`. |
-| 3D Samples | Sample scenes illustrating various ways to use Cinemachine in a 3D setting.<br />Includes: `Brain Update Modes`, `ClearShot`, `Custom Blends`, `Cutscene`, `FreeLook Deoccluder`, `FreeLook on Spherical Surface`, `Impulse Wave`, `Lock-on Target`, `MixingCamera`, `RunningRace`, `SplitScreenCar`, `ThirdPersonWithAimMode`, `ThirdPersonWithRoadieRun`. |
-| Input System Samples | Sample scenes illustrating various ways to use Cinemachine with the input system.<br />Includes: `Split Screen Multiplayer`. |
+### 2D Samples
+
+Once you import the 2D Samples set, the following scenes are available in the `Assets/Samples/Cinemachine/<version number>/2D Samples` folder.
+
+| Sample scene | Context | Use cases and key features |
+| :--- | :--- | :--- |
+| **2D Fighters** | Four-player fighting game (simulation). | <ul> <li>Set up a camera that dynamically composes the shot according to the remaining players in the game.</li> <li>Emulate players that move along predefined paths.</li> </ul> |
+| **2D Platformer** | 2D Platformer game with boss room. | <ul> <li>Set up a custom camera group that switches cameras based on the player state.</li> <li>Set up a camera that frames a boss room entirely when the player enters the room.</li> <li>Confine the camera to prevent it from seeing outside the game map.</li> </ul> |
+| **Camera Magnets** | 2D Platformer game with framing variations. | <ul> <li>Set up "magnets" that attract the camera's attention according to the player's proximity.</li> <li>Confine the camera to prevent it from seeing outside the game map.</li> </ul> |
+
+### 3D Samples
+
+Once you import the 3D Samples set, the following scenes are available in the `Assets/Samples/Cinemachine/<version number>/3D Samples` folder.
+
+| Sample scene | Context | Use cases and key features |
+| :--- | :--- | :--- |
+| **Brain Update Modes** | Car racing game (simulation). | <ul> <li>Set up the Cinemachine Brain and the animated objects in the camera composition to get a synchronized render of the whole.</li> </ul> |
+| **Clear Shot** | Third-person follow from various perspectives. | <ul> <li>Set up a group of fixed and moving cameras that follow the same player from different perspectives.</li> <li>Get the camera automatically switch to the best available shot when the current camera looses sight of the player.</li> </ul> |
```

Comment:
Summary: automatic selection of the best shot based on occlusion

Reply:
I can try to insert this in the 3rd column (according to my first comment).

Reply:
The fact that this is a third-person follow is completely irrelevant to the concept (and in fact only accurate at times).  Mentioning "third-person follow" is misleading, IMO.

Reply:
Yeah I know, that's why I need a 101 course from you about game types and camera types. Anyway here, the info will disappear as I'm removing the column.